.github/workflows/run-forge.yaml

```diff
@@ -113,40 +57,23 @@ jobs:
           PUSH_GATEWAY_PASSWORD: ${{ secrets.PUSH_GATEWAY_PASSWORD }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          set -x
-
-          # parse the JSON report
-          cat $FORGE_OUTPUT | awk '/====json-report-begin===/{f=1;next} /====json-report-end===/{f=0} f' > "${FORGE_REPORT}"
-          # If no report was generated, fill with default report
-          if [ ! -s "${FORGE_REPORT}" ]; then
-            echo '{"text": "Forge test runner terminated"}' > "${FORGE_REPORT}"
-          fi
-
-          # If no report text was generated, fill with default text
-          FORGE_REPORT_TXT=$(cat $FORGE_REPORT | jq -r .text)
-          if [ -z "$FORGE_REPORT_TXT" ]; then
-            FORGE_REPORT_TXT="Forge report text empty. See test runner output."
-          fi
-
-          if [ "$FORGE_EXIT_CODE" == "0" ]; then
-            FORGE_COMMENT_HEADER='### :white_check_mark: Forge test success'
-          else
-            FORGE_COMMENT_HEADER='### :x: Forge test failure'
-          fi
+          set +e
 
-          # remove the "aptos-" prefix to get the chain name as reported to Prometheus
-          FORGE_CHAIN_NAME=${FORGE_CLUSTER_NAME#"aptos-"} 
-          FORGE_DASHBOARD_LINK="${GRAFANA_BASE_URL}&var-namespace=${FORGE_NAMESPACE}&var-chain_name=${FORGE_CHAIN_NAME}&from=${FORGE_START_TIME_MS}&to=${FORGE_END_TIME_MS}"
+          # source this script to run and get environment variables
+          . scripts/run_forge.sh
```

Comment:
can't you actually use `source` ? The `.` looks kinda confusing

Reply:
sure -- just a bit hesitant sometimes since some distros don't have `source` :)